docs/Stream_Transformation_Functions.md

```diff
@@ -0,0 +1,52 @@
+# Stream transformation functions
+
+Create new stream transformation functions in Go. 
+
+## Required guidelines
+
+- Stream transformation functions must conform to the examples in [universe](https://github.com/influxdata/flux/blob/master/stdlib/universe) and include the required function and methods shown in the table below.
+- You must submit a unit test in in the same folder as the new stream transformation function implementation.
+- You must submit an end-to-end test in [testdata](https://github.com/influxdata/flux/tree/master/stdlib/testing/testdata). Please look at [End_to_End_Testing.md](https://github.com/influxdata/flux/tree/master/docs/End_to_End_Testing.md) for details.
+- You must add a description of the function to [SPEC.md](./docs/SPEC.md).
+
+### Required function and methods
+
+1. **init():**
+
+2. 1. Where you define your function signature and register the methods(**Op-Spec**, **Procedure Spec**, **Transformation Spec**)
+
+3. **FunctionSignature**: 
```

Comment:
- `FunctionSignature`: Defines how to write the function in Flux, including necessary inputs. For example, `timeShift`, specifies that duration and time columns must be included: 
```|>timeShift(duration: 10h, columns: ["_start", "_stop", "_time"])```